graphene_django/rest_framework/serializer_converter.py

```diff
@@ -138,3 +138,10 @@ def convert_serializer_field_to_jsonstring(field):
 @get_graphene_type_from_serializer_field.register(serializers.MultipleChoiceField)
 def convert_serializer_field_to_list_of_string(field):
     return (graphene.List, graphene.String)
```

Comment:
If serializers.ChoiceField is an Enum,
serializers.MultipleChoiceField would logically become (graphene.List, Enum)